server/src/main/java/com/datastrato/gravitino/server/web/ConfigServlet.java

```diff
@@ -44,12 +43,21 @@ public ConfigServlet(ServerConfig serverConfig) {
     }
   }
 
+  @Override
   protected void doGet(HttpServletRequest req, HttpServletResponse res)
-      throws ServletException, IOException {
+      throws java.io.UnsupportedEncodingException, IllegalStateException, IOException {
```

Comment:
Why is there a missing import for UnsupportedEncodingException?

Reply:
Sorry for the mistake. It's now fixed.